helm/APP-NAME/Chart.yaml

```diff
@@ -7,7 +7,6 @@ name: {APP-NAME}
 # Please describe what the app provides in a sentence or two.
 description: Please add description
 
-engine: gotpl
```

Comment:
Good to remove. There was an idea to support Lua templating too but it got dropped.
https://github.com/helm/helm/issues/5084